tests/test_keys/test_key_fragments.py

```diff
@@ -27,11 +28,7 @@ def test_cfrag_serialization_with_proof_and_metadata(alices_keys):
     kfrags = pre.split_rekey(priv_key_alice, pub_key_alice, 1, 2)
 
     # Example of potential metadata to describe the re-encryption request
-    metadata = { 'ursula_id' : 42, 
-                 'timestamp' : time.time(), 
-                 'capsule' : bytes(capsule), 
-               }
-    metadata = str(metadata).encode()
+    metadata = b'This is an example of metadata for re-encryption request' 
```

Comment:
Is there anything more useful we can put in here that may demonstrate its utility? This can be NuCypher agnostic, too.

Reply:
Sure, but we went over this before and we ended up with this again.

Reply:
But maybe we can demonstrate using a timestamp or  perhaps even the capsule itself?

Reply:
Yes, I did that in #108 but it was decided in #115 to revert to the simple approach.

See https://github.com/nucypher/pyUmbral/pull/115#discussion_r183594166